app/jsx/pages/CampusPage.jsx

```diff
@@ -0,0 +1,105 @@
+// ##### Campus Page ##### // 
+// Temporary until UNIT-CONTENT refactoring is complete // 
+
+import React from 'react'
+import { Link } from 'react-router'
+
+import PageBase from './PageBase.jsx'
+import Header2Comp from '../components/Header2Comp.jsx'
+import SubheaderComp from '../components/SubheaderComp.jsx'
+import NavComp from '../components/NavComp.jsx'
+import CampusSearchComp from '../components/CampusSearchComp.jsx'
+import CampusSelectorComp from '../components/CampusSelectorComp.jsx'
+import HeatMapComp from '../components/HeatMapComp.jsx'
+import CampusCarouselComp from '../components/CampusCarouselComp.jsx'
+
+class CampusPage extends PageBase
+{
+  // PageBase will fetch the following URL for us, and place the results in this.state.pageData
+  pageDataURL() {
+    return "/api/unit/" + this.props.route.path.split('/')[2] + "/campus_landing"
+  }
+
+  dashUrlList = {
+    'ucb': 'https://dash.berkeley.edu/stash',
+    'uci': 'https://dash.lib.uci.edu/stash',
+    'ucm': 'https://dash.ucmerced.edu/stash',
+    'ucop': 'https://dash.ucop.edu/stash',
+    'ucr': 'https://dash.ucr.edu/stash',
+    'ucsc': 'https://dash.library.ucsc.edu/stash',
+    'ucsf': 'https://datashare.ucsf.edu/stash',
+  }
```

Comment:
Eventually the dash URL should be an attr on the entity that's configured on the campus CMS page. But this is fine for now.